apps/api/v2/src/ee/overlay-calendars/controllers/overlay-calendars.controller.ts

```diff
@@ -0,0 +1,57 @@
+import { GetOverlayCalendarsBusyTimesInput } from "@/ee/overlay-calendars/inputs/get-overlay-calendars-busy-times.input";
+import { OverlayCalendarsService } from "@/ee/overlay-calendars/services/overlay-calendars.service";
+import { GetUser } from "@/modules/auth/decorators/get-user/get-user.decorator";
+import { AccessTokenGuard } from "@/modules/auth/guards/access-token/access-token.guard";
+import { Controller, Get, Logger, UseGuards, Body } from "@nestjs/common";
+import { User } from "@prisma/client";
+
+import { SUCCESS_STATUS } from "@calcom/platform-constants";
+import { ApiResponse } from "@calcom/platform-types";
+import { EventBusyDate } from "@calcom/types/Calendar";
+
+@Controller({
+  path: "ee/overlay-calendars",
```

Comment:
Why the `ee` prefix for path? I think that some endpoints (non ee folder) not having ee in front and others having might be more confusing developer experience unless I am missing something.

Reply:
I think all the endpoints in ee should have this prefix to avoid confusion